cmd/tt/buckets.go

```diff
@@ -203,22 +203,22 @@ File structure is mirrored in the bucket. For example, given the directory:
 
 These 'push' commands result in the following bucket structures.
 
-'textile buckets push foo mybuck':
+'tt buckets push foo mybuck':
     mybuck/foo/one.txt
     mybuck/foo/bar/two.txt
     mybuck/foo/bar/baz/three.txt
 
-'textile buckets push foo/bar mybuck':
+'tt buckets push foo/bar mybuck':
     mybuck/bar/two.txt
     mybuck/bar/baz/three.txt
 
-'textile buckets push foo/bar/baz mybuck':
+'tt buckets push foo/bar/baz mybuck':
     mybuck/baz/three.txt
 
-'textile buckets push foo/bar/baz/three.txt mybuck':
+'tt buckets push foo/bar/baz/three.txt mybuck':
     mybuck/three.txt
 
-'textile buckets push foo/* foo':
+'tt buckets push foo/* foo':
```

Comment:
If I understand the latest PR, this one will need to change too. But I'll do it once I work through buckets fully. I hit a snag last night with PBs out of sync.

Reply:
I'm gonna revamp the push and pull anyway so can update the docs then